scripts/manage_study.py

```diff
@@ -266,19 +298,31 @@ def login(manager=None, dry_run=False):
                                     study_name=parsed_args.study_name,
                                     cluster_name=parsed_args.cluster_name,
                                     description=parsed_args.cluster_description,
-                                    species=parsed_args.species,
-                                    genome=parsed_args.genome,
                                     x=parsed_args.x_label,
                                     y=parsed_args.y_label,
                                     z=parsed_args.z_label,
                                     dry_run=parsed_args.dry_run)
     manage_call_return(ret)
-'''
+
 ## Upload metadata file
-### TODO
+if hasattr(parsed_args, "metadata_file"):
+    print("UPLOAD METADATA FILE")
```

Comment:
I think a future enhancement could be to make these logging statements a little more informative.  I'm thinking something like this:

#{timestamp}: Uploading #{file_type} to #{study_name}